src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -74,25 +74,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 			response.sendRedirect("/index.html");
 			return;
 		}
-
-		String user = userService.getCurrentUser().getEmail();
+    String user = userService.getCurrentUser().getEmail();
 		String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 		
+
 		Message message = new Message(user, text);
 		datastore.storeMessage(message);
-
-    Message message = new Message(user, text);
-    datastore.storeMessage(message);
-
+    
     String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
     
     String regex = "(https?://\\S+\\.(png|jpg))";
     String replacement = "<img src=\"$1\" />";
     String textWithImagesReplaced = userText.replaceAll(regex, replacement);
-        
+      /* 
     message = new Message(user, textWithImagesReplaced);
     datastore.storeMessage(message);
-
+    */
     response.sendRedirect("/user-page.html?user=" + user);//this goes into the url 
   }
 }
```

Comment:
I think this file is also outdated because it doesn't include a lot of the changes that are present in the master.